package.json

```diff
@@ -92,6 +97,8 @@
     "@types/prettier": "2.7.1",
     "@types/shell-quote": "1.7.1",
     "@your-org/eslint-config-bases": "workspace:^",
+    "commitizen": "4.2.5",
+    "cz-conventional-changelog": "3.3.0",
```

Comment:
Does it competes with changeset ?

Reply:
sorry forgot generate an changeset for  package.json update BTW in my locall `main` is latest code, `yarn dedupe --check` nothing changed of `.yarn.lock`

Reply:
BTW the yarn g:changeset cause of updated root/package.json i could not choose correct package for this.

<img width="466" alt="image" src="https://user-images.githubusercontent.com/2833251/195330970-327648e0-ce4e-49fd-be16-776ef3803c0a.png">

nothing can indicates `./root` how to do this?

Reply:
My question was not clear I think :) I was confused and wrongly thought that `cs-conventional-changelog` was generating the changelogs. But it's not. Sorry.

About changesets, there's no root concept. I recommend in this to add a 'patch' to all packages (I mean it's more for keep a doc)

But you don't have too. I'll resolve the comment. Nothing to do